docs/UserGuide.md

```diff
@@ -357,31 +493,42 @@ and zoom in to your guest list accordingly.
 
 Format: `guest filter [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/RSVP_STATUS] [tn/TABLE_NUMBER] [d/DIETARY_REQUIREMENT]… [t/TAG]…`
 
->Tips:
-><br>
->- Parameters in square brackets are optional. However, you must include at least one parameter to filter by.
-   ><br>
->- For all parameters aside from NAME and RSVP_STATUS, providing an empty value will filter for guests with unfilled values for that parameter.
-   ><br>
->- Providing an empty `r/` value would filter for guests with unknown RSVP status.
-   ><br>
->- Providing an empty `n/` value would filter for guests without a name thus returning an empty guest list. (All guests need a name)
-   ><br>
->- The filter command will only return guests that matches all the input values across different parameters.
-   ><br>
->- DIETARY_REQUIREMENT and TAG parameters can be inputted multiple times (e.g. "guest filter d/no beef d/no pork"). 
-   However, do take note that these two parameters are filtered via a case-insensitive exact match (i.e. a guest with the tag "friends" would not be a valid result for the input "t/friend").
-   ><br>
->- TABLE_NUMBER and RSVP_STATUS parameter are also filtered via a case-insensitive exact match (i.e. `guest filter r/n` will not return guests with rsvp status `no`)
->- The rest of the parameters (NAME, PHONE, EMAIL, ADDRESS) are filtered via a case-insensitive partial match (i.e. "guest filter n/john" returns "john" and "johnathan"). 
->- Acceptable values for the parameters can be any number of alphanumeric characters for all parameters.
+<div class="alert alert-block alert-warning">
+
+**Parameter information:**
+
+* You can use any combination of letters, numbers and special characters as inputs for all parameters. 
+* Parameters in square brackets are optional. However, you must include at least one parameter to filter by.
+* For all parameters aside from `NAME` and `RSVP_STATUS`, providing an empty value (e.g. `p/`) will filter for guests with unfilled values for that parameter. 
+* Providing an empty `r/` value would filter for guests with RSVP status `Unknown`. 
+* Providing an empty `n/` value would filter for guests without a name thus returning an empty guest list. (All guests need a name)
```

Comment:
Let's standardize whether we refer to the fields using the name or the label. I think the name is more clear.

```suggestion
* For all parameters aside from `NAME` and `RSVP_STATUS`, providing an empty value (e.g. `p/`) will filter for guests with unfilled values for that parameter. 
* Providing an empty `RSVP_STATUS` value would filter for guests with RSVP status `Unknown`. 
* Providing an empty `NAME` value would filter for guests without a name, thus returning an empty guest list as all guests need a name.
```